README.md

```diff
@@ -1 +1,123 @@
-# covid-19-data
\ No newline at end of file
+# COVID Data Hub | MODULE-III
+
+> Covid-data is simple application being built in our third module of our curriculum at microverse, where we build a web application which provides a list of countires with COVID summary. The Details page shows the detailed COVID data of each country selected.
+
+## screen shots
+
+![screenshot](/screenshots/screen1.png)
+![screenshot](/screenshots/screen2.png)
+
+### Features implemented so far are:
+
+- Responsive HTML page
+- Set up a React app.
+- Use React documentation.
+- Use React documentation.
+- Use React components.
+- Use React props.
+- Use React Router.
+- Connect React and Redux.
+- Handle events in a React app.
+- Write unit tests with React Testing Library.
+- Use styles in a React app.
+- Use React life cycle methods.
+- Apply React best practices and language style guides in code.
+- Use store, actions and reducers in React.
+
+## Built With
+
+- REACT
+- REDUX
+- Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
+- ESlint (A mighty, modern linter that helps you avoid errors and enforce conventions in JavaScript codes)
+- Webpack (A JavaScript module bundler)
+
+To get a local copy up and running follow these simple example steps.
+
+[Github-page live](https://L2oukmane.github.io/covid-19-data)
+
+### Prerequisites
+
+The basic requirements for building the executable are:
+
+- A working browser application (Google chrome, Mozilla Fire fox, Microsoft edge ...)
+- VSCode or any other equivalent code editor
+- Node Package Manager (For installing packages like Lighthous, webhint & stylelint used for checking for debugging bad codes before deployment)
+
+#### Acknowledgments
+
+- Inspired by original design for Creative Commons by Nelson Sakwa on [Behance](<https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)>)
```

Comment:
Well done giving appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) 🤓